src/directives/popover.js

```diff
@@ -0,0 +1,137 @@
+goog.provide('ngeo.popoverDirective');
+goog.provide('ngeo.popoverAnchorDirective');
+goog.provide('ngeo.popoverContentDirective');
+goog.provide('ngeo.PopoverController');
+
+goog.require('ngeo');
+
+
+/**
+ * Provides a directive used to display a Bootstrap popover.
+ *
+ * @example <caption> Basic usage of nego popover </caption>
+ *<div ngeo-popover>
+ *  <a ngeo-popover-anchor class="btn btn-info">anchor 1</a>
+ *  <div ngeo-popover-content="">
+ *    <ul>
+ *      <li>action 1:
+ *        <input type="range"/>
+ *      </li>
+ *    </ul>
+ *  </div>
+ *</div>
+ * @ngdoc directive
+ * @ngInject
+ * @ngname ngeoPopover
+ * @return {angular.Directive} The Directive Definition Object.
+ */
+ngeo.popoverDirective = function() {
+  return {
+    restrict: 'A',
+    scope: {},
+    controller: 'NgeoPopoverController',
+    link: function(scope, elem, attrs, controller) {
+      function onClick(clickEvent) {
```

Comment:
put the logic in the controller for more consistency (remove the `link` function).